spec/features/donor_edits_location_spec.rb

```diff
@@ -8,7 +8,6 @@
     new_pickup_location = {
       address: "123 Fake Street Denver CO",
       zipcode: zone.zipcode,
-      notes: "on my porch",
     }
```

Comment:
@desi temporarily.

I've made [a note on the card](https://trello.com/c/Oz6iPKen/78-as-a-supplier-i-want-to-be-able-to-log-into-the-app-and-view-my-profile#comment-570d75cb2e569299dd8b6f79).

The idea is that it's not currently working properly, and needs some design. My options were either to remove it and get tests passing, make the tests pass by injecting it into the page without regard for design, or block the PR until the design is ready.


Do you think we should keep it around?